reth/Dockerfile

```diff
@@ -0,0 +1,49 @@
+FROM golang:1.21 AS op
+
+WORKDIR /app
+
+ENV REPO=https://github.com/ethereum-optimism/optimism.git
+ENV VERSION=v1.7.7
+ENV COMMIT=f8143c8cbc4cc0c83922c53f17a1e47280673485
+RUN git clone $REPO --branch op-node/$VERSION --single-branch . && \
+    git switch -c branch-$VERSION && \
+    bash -c '[ "$(git rev-parse HEAD)" = "$COMMIT" ]'
+
+RUN cd op-node && \
+    make VERSION=$VERSION op-node
+
+FROM rust:1.79 AS reth
+
+ARG FEATURES=jemalloc,asm-keccak,optimism
+
+WORKDIR /app
+
+RUN apt-get update && apt-get -y upgrade && apt-get install -y git libclang-dev pkg-config curl build-essential
+
+ENV REPO=https://github.com/paradigmxyz/reth.git
+ENV VERSION=v1.0.0
+ENV COMMIT=83d412da70af678a46f368533b6df45a287a1ce6
+RUN git clone $REPO --branch $VERSION --single-branch . && \
+    git switch -c branch-$VERSION && \
+    bash -c '[ "$(git rev-parse HEAD)" = "$COMMIT" ]'
+
+RUN cargo build --bin op-reth --locked --features $FEATURES --profile release
```

Comment:
Let's create an issue to update the profile to `maxperf` for when it starts working properly again.

Reply:
Created https://github.com/LiskHQ/lisk-node/issues/9